packages/libp2p-daemon/package.json

```diff
@@ -136,11 +136,25 @@
     "release": "aegir release"
   },
   "dependencies": {
-    "@libp2p/daemon-server": "^3.0.0",
+    "@chainsafe/libp2p-gossipsub": "^4.1.1",
+    "@chainsafe/libp2p-noise": "^8.0.1",
+    "@libp2p/bootstrap": "^2.0.0",
+    "@libp2p/connection": "^4.0.1",
+    "@libp2p/crypto": "^1.0.4",
+    "@libp2p/daemon-server": "^2.0.0",
```

Comment:
why did this version go from 3.0.0 to 2.0.0?

Reply:
Not sure what happened there, it is now fixed.